music_rnn/src/model.ts

```diff
@@ -52,6 +52,8 @@ export class MusicRNN<A extends magenta.controls.ControlSignalUserArgs> {
    * `NoteSequence` and `Tensor` objects. If not provided, a `converter.json`
```

Comment:
Can you add an example to the demo either here or in a follw-up PR?

Reply:
I'll do another PR where I add examples with chord-conditioning in both music_rnn and music_vae.